src/js_native_api_v8.cc

```diff
@@ -1602,13 +1602,10 @@ napi_status napi_create_bigint_words(napi_env env,
   v8::MaybeLocal<v8::BigInt> b = v8::BigInt::NewFromWords(
       context, sign_bit, word_count, words);
 
-  if (try_catch.HasCaught()) {
-    return napi_set_last_error(env, napi_pending_exception);
-  } else {
-    CHECK_MAYBE_EMPTY(env, b, napi_generic_failure);
-    *result = v8impl::JsValueFromV8LocalValue(b.ToLocalChecked());
-    return napi_clear_last_error(env);
-  }
+  CHECK_MAYBE_EMPTY_WITH_PREAMBLE(env, b, napi_generic_failure);
```

Comment:
No tests - no pull-request

Reply:
The [new test](https://github.com/nodejs/node/pull/34554/files#diff-2ceae2fb7a63e3242c23e5378a2c54ccR48) included with this PR tests the correct behaviour of `CHECK_MAYBE_EMPTY_WITH_PREAMBLE()`.